### ISSUE TITLE
128 channel headstage

### ISSUE THREAD

**Issue by teresajurado** [open-ephys/acquisition-board#20]
Dear all, 
It is compatible the open ephys adquition board https://open-ephys.org/acq-board with the intan RHD 128-channel headstage https://intantech.com/RHD_headstages.html?tabSelect=RHD128ch&yPos=0 ?

Thanks!

**Comment by jsiegle** [comment]
Yes, this headstage is fully supported. It will appear as two 64-channel headstages in the acquisition board plugin.
